### ISSUE TITLE
Almost invisible links in PM (suckeberg theme)

### ISSUE THREAD

**Issue by hfroese** [DeadSuperHero/hubzilla-themes#20]
Links in PM are almost invisble

![20180311-0900](https://user-images.githubusercontent.com/1968490/37250961-0ded474c-2508-11e8-86bb-38f8781b32c8.png)

**Comment by DeadSuperHero** [comment]
Good catch, I'll look into fixing this in a sane way. Probably might be
better to just use a different background color, I guess.
